.github/ISSUE_TEMPLATE/recommend_guest.yaml

```
@@ -0,0 +1,85 @@
+name: 推荐嘉宾
+description: 如果您想推荐一位嘉宾录制播客，请选择本条模板填写内容
+title: '推荐嘉宾 - [姓名/花名/昵称]（栏目）'
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### 标题格式
+        推荐格式为 `推荐嘉宾 - [推荐人的姓名/花名/昵称] （栏目）`，栏目可省略；
+        例如：`推荐嘉宾 - 张三（个人成长系列）` `推荐嘉宾 - 李四`
+
+  - type: input
+    id: channel
+    attributes:
+      label: 栏目
+      description: |
+        目前已有栏目 `个人成长系列`、`开源岗位系列`、`开源项目系列`、`番外篇`。
+        最新栏目设置请移步 [访谈系列](https://github.com/opensource-f2f/episode#%E8%AE%BF%E8%B0%88%E7%B3%BB%E5%88%97) 查看。
+    validations:
+      required: true
+
+  - type: input
+    id: name
+    attributes:
+      label: 嘉宾名称
+      description: 姓名、花名或昵称
+    validations:
+      required: true
+
+  - type: input
+    id: github
+    attributes:
+      label: GitHub
+      description: GitHub 链接，可选
+
+  - type: input
+    id: org
+    attributes:
+      label: 公司/组织/岗位
+      description: 所在公司、组织或岗位，可选
+
+  - type: input
+    id: twitter
+    attributes:
+      label: Twitter
+      description: Twitter ID，可选
+
+  - type: input
+    id: blog
+    attributes:
+      label: 博客地址
+      description: 或其他媒体平台，可选
+
+  - type: textarea
+    id: reason
+    attributes:
+      label: 推荐理由
+    validations:
+      required: true
+
+  - type: input
+    id: referrer
+    attributes:
+      label: 推荐人
+    validations:
+      required: true
+
+  - type: checkboxes
+    attributes:
+      label: note
+      options:
+        - label: 建议嘉宾 [订阅《开源面对面》的邮件列表](mailto:osf2f+subscribe@googlegroups.com)（发送邮件被视为订阅），方便后续发送录制的会议邀请。
```

```suggestion
        - label: 建议嘉宾 [订阅《开源面对面》的邮件列表](https://groups.google.com/g/osf2f)（发送邮件被视为订阅），方便后续发送录制的会议邀请。
```
替换成 `https://groups.google.com/g/osf2f` 会不会好一些

---

https://groups.google.com/g/osf2f 这个地址，有的人可能不太方便访问，但发送邮件没有限制。